portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/TranslationManager.path

```diff
@@ -0,0 +1,66 @@
+<html>
+<head>
+<title>TranslationManager</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">TranslationManager</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>SELECT_TRANSLATION_LANGUAGE_MENU</td>
+	<td>//div[h3='${key_title}']//button[contains(@title,'select-translation-language')]//*[name()='svg'][contains(@class,'lexicon-icon-${key_currentLocale}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>SELECT_TRANSLATION_LANGUAGE_AUI_TAG_ADMINMENU</td>
+	<td>//button[contains(@id,'aui_2d_2Menu')]//*[name()='svg'][contains(@class,'lexicon-icon-${key_currentLocale}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>SELECT_TRANSLATION_LANGUAGE_MENU_ITEM_INDEX</td>
+	<td>//div[contains(@class,'dropdown-menu') and contains(@class,'show')]//ul[contains(@class,'list-unstyled')]//li[${key_index}]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>MANAGE_TRANSLATIONS</td>
+	<td>//button[contains(@class,'dropdown-item')]//div[contains(@class,'autofit-section')and text()='Manage Translations']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>MANAGE_TRANSLATIONS_DIALOG_SEARCH_FIELD</td>
+	<td>//div[contains(@class,'modal-dialog')]//input[contains(@placeholder,'Search')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>MANAGE_TRANSLATIONS_DIALOG_SEARCH_RESULTS_TITLE</td>
+	<td>//th[contains(text(),'${key_text}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>MANAGE_TRANSLATIONS_DIALOG_LANGUAGE_LIST</td>
+	<td>//tr[${key_index}]//*[name()='svg'][contains(@class,'lexicon-icon-${key_locale}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>MANAGE_TRANSLATIONS_DIALOG_LANGUAGE_DELETE</td>
+	<td>//tr[${key_index}]//*[name()='svg'][contains(@class,'lexicon-icon-trash')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>AUI_TAG_INPUT_TEXT_FIELD</td>
+	<td>//div[contains(@class,'col') and h3='${key_title}']//input[contains(@class,'language-value field')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>AUI_TAG_BELOW_INPUT_TEXT_FIELD</td>
```

Comment:
This name still doesn't look good to me. The field actually contains the default language text after selecting another language. I think it's better to describe the element with something like "default_language_value" or "default_language_text". But `BELOW_INPUT_TEXT_FIELD` doesn't look like an element for me, it's more like the whole area that below the input field.

Reply:
I edit it, and it is named AUI_TAG_DEFAULT_LANGUAGE_TEXT now.